pkg/ctl/enable/profile.go

```diff
@@ -94,6 +94,9 @@ func doEnableProfile(cmd *cmdutils.Cmd, opts *ProfileOptions) error {
 		return err
 	}
 	usersRepoDir, err := ioutil.TempDir("", usersRepoName+"-")
+	if err != nil {
+		return err
```

Comment:
It's good to add some more context to the error like:
```
        return errors.Wrapf(err, "unable to create temporary directory for %q", usersRepoName)
```

Reply:
Yeah right, thanks for your input. I have updated the same.